helios-client/src/main/java/com/spotify/helios/common/descriptors/Job.java

```diff
@@ -188,6 +190,8 @@
    * @param addCapabilities Linux capabilities to add for the container. Optional.
    * @param dropCapabilities Linux capabilities to drop for the container. Optional.
    * @see <a href="https://docs.docker.com/reference/run/#network-settings">Docker run reference</a>
+   * @param secondsToWaitBeforeKill The time to ask Docker to wait after sending a SIGTERM to the
+   *    container's main process before sending it a SIGKILL. Optional. Defaults to 120s.
```

Comment:
nitpick but it does not default to 120s in regards to the value of the field in this Job class. The agent will default to using 120s if the Job. secondsToWaitBeforeKill is empty/null. I rather leave the value undocumented in this class so we can change the agent if we have to later without changing this part (so it's really just an issue of moving the documentation).